Prelude/BooleanType.swift

```diff
@@ -1,11 +0,0 @@
-public func isTrue(b: BooleanType) -> Bool {
```

Comment:
no more `BooleanType` protocol so just moved to https://github.com/kickstarter/Kickstarter-Prelude/blob/277203de6203b9288f7add85e5b2c0d2668829da/Prelude/Boolean.swift